agent-jenkins-tools/Dockerfile

```diff
@@ -3,42 +3,29 @@ FROM quay.io/feedhenry/jenkins-agent-java:latest
 USER root
 
 # Jenkins Job Builder
-
 ENV BRANCH master
 
 WORKDIR /
-RUN git clone -b allow_self_signed_certs https://github.com/mikenairn/python-jenkins.git && \
-    pushd python-jenkins && \
-    yum -y --enablerepo=extras install epel-release && \
+RUN yum -y --enablerepo=extras install epel-release && \
     yum -y install python-pip skopeo && \
     yum clean all && \
     rm -rf /var/cache/yum && \
-    pip install -U -r requirements.txt && python setup.py install && \
-    popd && \
-
+    \
```

Comment:
@pmccarthy I like the empty lines for readability, so I've added these line continuations to hopefully stop docker complaining. Could you try building locally and see if you still see those warnings?

Reply:
@grdryn Yeah that seems to work fine. Not seeing those warnings anymore